lib/byron/test/data/cardano-node/node.config

```diff
@@ -0,0 +1,73 @@
+# vim: set filetype=yaml:
+# -*- mode: yaml -*-
+
+#     _   _           _          ____             __ _
+#    | \ | | ___   __| | ___    / ___|___  _ __  / _(_) __ _
+#    |  \| |/ _ \ / _` |/ _ \  | |   / _ \| '_ \| |_| |/ _` |
+#    | |\  | (_) | (_| |  __/  | |__| (_) | | | |  _| | (_| |
+#    |_| \_|\___/ \__,_|\___|   \____\___/|_| |_|_| |_|\__, |
+#                                                      |___/
+
+NodeId:
+NumCoreNodes: 1
+PBftSignatureThreshold: 1000000000000000
+Protocol: RealPBFT
+RequiresNetworkMagic: RequiresMagic
+TurnOnLogMetrics: False
+TurnOnLogging: True
+ViewMode: SimpleView
+
+
+#     _   _           _       _          ____
+#    | | | |_ __   __| | __ _| |_ ___   |  _ \ __ _ _ __ __ _ _ __ ___  ___
+#    | | | | '_ \ / _` |/ _` | __/ _ \  | |_) / _` | '__/ _` | '_ ` _ \/ __|
+#    | |_| | |_) | (_| | (_| | ||  __/  |  __/ (_| | | | (_| | | | | | \__ \
+#     \___/| .__/ \__,_|\__,_|\__\___|  |_|   \__,_|_|  \__,_|_| |_| |_|___/
+#          |_|
+
+ApplicationName: cardano-sl
```

Comment:
really this application name?

Reply:
Ah! Good point :thinking: ... I _think_ it does not really matter here since this is our integration configuration, so its a private blockchain. It could be anything we like :) 

(as for where this comes from, I just shamelessly copied it from: https://github.com/input-output-hk/cardano-node/blob/master/benchmarking/cluster3nodes/configuration/log-config-0.yaml)